cmake/android/tester/CMakeLists.txt.in

```diff
@@ -64,7 +60,7 @@ target_link_libraries(${TESTER_LIB}
     olp-cpp-sdk-core
     ${SVC_LIB}
     ${SVC_TEST_LIB}
-    custom-params
+    gtest
```

Comment:
Only gtest? No gmock?

Reply:
That's the tests' runner library, not the library with actual tests. So, it only needs to invoke the `RUN_ALL_TESTS` method. Currently there are no mocks there, so, I think, `gtest` is enough.